Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.54.0
```

Comment:
Should this maybe be a separate PR?

Reply:
This PR fails ci without bumping this - I don't see the benefit of splitting out a change like this when it's required (especially given githubs.. poor support of dependent changes vs. e.g. gerrit)